src/tests/multihost/alltests/test_sss_cache.py

```diff
@@ -0,0 +1,60 @@
+""" Automation of sss_cache tests
+
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+import ldap
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from sssd.testlib.common.expect import pexpect_ssh
+from sssd.testlib.common.exceptions import SSHLoginException
+
+
+@pytest.mark.usefixtures('setup_sssd_krb', 'create_posix_usersgroups')
+@pytest.mark.sss_cache
+class TestKrbWithLogin(object):
```

Comment:
It will be better to use a different class name. This is same as
https://github.com/SSSD/sssd/blob/master/src/tests/multihost/alltests/test_krb5.py#L15

Reply:
Done